packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.1-nullsafety
```

Comment:
Must be `2.0.0-nullsafety.1` as the same value already declared in pubspec.

Reply:
oh yeah, sorry 😅 completely forgot about that, very sorry abt that